app/views/splash/index.html.erb

```diff
@@ -5,10 +5,8 @@
 
     </div>
     <div class="flex flex-col justify-end h-1/2 w-full">
-      <%= link_to "Sign In", new_user_session_path, class:"text-white h-12 bg-main from-green-400 via-green-500 to-green-600 hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-green-300 dark:focus:ring-green-800 font-medium rounded-sm text-sm px-5 py-2.5 text-center mr-2 mb-2"%>
-      <%= link_to "Sign Up", new_user_registration_path, class:"text-white h-12 bg-main from-green-400 via-green-500 to-green-600 hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-green-300 dark:focus:ring-green-800 font-medium rounded-sm text-sm px-5 py-2.5 text-center mr-2 mb-2"%>
-
-
+      <%= link_to "Sign In", new_user_session_path, class:" w-full text-white h-12 bg-main from-green-400 via-green-500 to-green-600 hover:bg-gradient-to-br focus:ring-4 focus:outline-none focus:ring-green-300 dark:focus:ring-green-800 font-medium rounded-sm text-sm px-5 py-2.5 text-center mr-2 mb-2"%>
```

Comment:
- Please make sure the login page looks like the mockup
## Your login page
![Screenshot from 2022-11-20 00-25-05](https://user-images.githubusercontent.com/80149314/202873820-af73033d-7e2e-4613-b5e4-b0634df89ce1.png)

## mockup login page
![Screenshot from 2022-11-20 00-22-03](https://user-images.githubusercontent.com/80149314/202873906-e5a7a24a-9509-4500-a84e-6570d8dcf609.png)